app/controllers/v0/addresses_controller.rb

```diff
@@ -41,26 +41,14 @@ def states
              serializer: StatesSerializer
     end
 
-    def rds_countries
-      response = strategy.cache_or_service(:countries) { rds_service.get_countries }
-      render json: response,
-             serializer: CountriesSerializer
-    end
-
-    def rds_states
-      response = strategy.cache_or_service(:states) { rds_service.get_states }
-      render json: response,
-             serializer: StatesSerializer
-    end
-
     private
 
     def service
-      EVSS::PCIUAddress::Service.new(@current_user)
-    end
-
-    def rds_service
-      EVSS::ReferenceData::Service.new(@current_user)
+      if Settings.evss&.reference_data_service&.enabled
```

Comment:
`Settings.evss.reference_data_service.enabled` 

Will `Settings.evss` ever be nil? Not sure we need the first guard clause

Reply:
I think you're referring to the safe navigator, right(`&.`)?

Ya, probably not, I'll remove it. If `Settings.evss` was ever `nil` I think this class would be the least of our problems 😄 .